AK/NonnullOwnPtr.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <AK/Assertions.h>
+#include <AK/Badge.h>
```

Comment:
I don't think this is a good use case for this optimization, if you want to have a smaller `Optional<NonnullRefPtr>` just use a `RefPtr`. Especially as this optimization doesn't come for free and adds quite a lot of complexity in the optimized `Optional` implementation (`dont_move_empty` and `has_swap`).

Reply:
OK will drop it then